pegasus/sites.v3/code.org/public/index2.haml

```diff
@@ -64,6 +64,24 @@ social:
     - opacity = index == 0 ? 1 : 0
     .widehero{id: "fullwidth#{index}", style: "display: #{display}; z-index: -5; position:absolute; left: 0; top: 0; background-image: url(#{entry[:image_t]}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: #{opacity}; background-position: 50% 50%"}
 
+
+  -facebook = {:u=>"http://#{request.site}/"}
+
+  -if request.site == 'csedweek.org'
+    -twitter = {:related=>'codeorg', :hashtags=>'', :text=>I18n.t(:hoc_is_here)}
+    -twitter[:hashtags] = 'HourOfCode' unless I18n.t(:hoc_is_here).include? '#HourOfCode'
+  -else
+    -twitter = {:related=>'codeorg', :hashtags=>'', :text=>"Why don't more schools teach computer science? We can fix this. http://youtu.be/nKIu9yen5nc. Visit http://code.org/ to learn more."}
```

Comment:
This should be localized too. I added to i18n gsheet with key 'twitter_teach_cs'